lib/services/gitlab_issues.rb

```diff
@@ -0,0 +1,323 @@
+class AhaServices::GitlabIssues < AhaService
+    title 'GitLab Issues'
+    caption 'Send features to GitLab Issues'
+
+    password :private_token
+    string :server_url, description: 'If you are using your own GitLab server please enter your server URL without a trailing slash (https://example.com/api/v3). If you are using gitlab.com leave this field empty.',
+                        label: 'Server URL'
+    install_button
+    select :project, collection: -> (meta_data, _data) do
+        return [] if meta_data.nil? || meta_data.repos.nil?
+        meta_data.repos.sort_by(&:name).collect { |repo| [repo.full_name, repo.full_name] }
+    end
+
+    select :mapping, collection: [
+        ['Feature -> Issue, Requirement -> Issue', 'issue-issue'],
+        ['Feature -> Issue, Requirement -> Checklist item', 'issue-checklist']
+    ], description: 'Choose how features and requirements in Aha! will map to issues and checklists in GitLab.'
+
+    string :due_date_phase, description: 'The name of the phase to find the due date of a release', label: 'Due Date Phase'
```

Comment:
In GitLab this is just a due date for the Milestone that we create from the release.  In this case the client requested that the end date on the phase named "Development" should flow over to the milestone.  If there is no Development phase we're using the release date instead, which is the same date used by the GitHub integration.